controllers/pvc.go

```diff
@@ -27,6 +27,12 @@ func (r *VolumeSnapshotBackupReconciler) MirrorPVC(log logr.Logger) (bool, error
 		return false, err
 	}
 
+	// no need to mirror pvc for the vsb if the datamovement has already completed, no need to fetch source VS as it will be deleted once backup completes
+	if len(vsb.Status.Phase) > 0 && vsb.Status.Phase == volsnapmoverv1alpha1.SnapMoverVolSyncPhaseCompleted {
```

Comment:
Should `vsb.Status != nil` be checked here first to prevent nil pointer issues?

Reply:
we cannot do that because vsb status is not a pointer hence the string len check for phase directly